### DISCUSSION TITLE
Becoming aware about the outcome of this course

### DISCUSSION THREAD

**Discussion by vigneshk004** [Cyfrin/foundry-full-course-cu#2688] [General]
Will I be able to create my own cryptocurrency after completing all the courses....????

**Comment by EngrPips** [comment]
If you understand what you are taught, yes!!!!!!!!!!!!!!!

**Reply by vigneshk004** [reply]
Thank you, has anybody who have completed this course tried that.....?????

**Reply by EngrPips** [reply]
I don't know, maybe they will see this discussion and respond to you.

**Reply by vigneshk004** [reply]
Alright, well thank you..for reaching out back to me and answering my query..

**Reply by EngrPips** [reply]
You are welcome.

**Comment by BowtiedHarpyEagle** [comment]
I'm going through the stable coin part of the course, so if that's your goal, you could do that.
